packaging/tests/tas-restapi/src/test/java/org/alfresco/rest/rules/UpdateRulesTests.java

```diff
@@ -198,6 +200,27 @@ public void updateRuleWithIncludedFields()
         updatedRule.assertThat().field("isShared").isNotNull();
     }
 
+    /** Check we can use the POST response to create the new rule. */
+    @Test (groups = { TestGroup.REST_API, TestGroup.RULES, TestGroup.SANITY })
+    public void updateCopyRuleWithResponseFromPOST()
+    {
+        FolderModel destination = dataContent.usingUser(user).usingSite(site).createFolder();
+
+        RestActionBodyExecTemplateModel copyAction = new RestActionBodyExecTemplateModel();
+        copyAction.setActionDefinitionId("copy");
+        copyAction.setParams(ImmutableMap.of("destination-folder", destination.getNodeRef()));
+        RestRuleModel rule = createAndSaveRule("Rule name", List.of(copyAction));
+
+        STEP("Try to update the rule.");
+        rule.setName("Updated rule name");
+        RestRuleModel updatedRule = restClient.authenticateUser(user).withCoreAPI().usingNode(ruleFolder).usingDefaultRuleSet()
+                                              .include("isShared")
+                                              .updateRule(rule.getId(), rule);
+
+        restClient.assertStatusCodeIs(OK);
+        updatedRule.assertThat().field("name").is("Updated rule name");

```

Comment:
Should we also check that `updatedRule` also has an action parameter `actionContext->rule`?
Or even we could do a similar check to the one we do in `CreateRulesTests`:
`updatedRule.assertThat().isEqualTo(rule, <list of ignored fields if needed>);`